Build/_build.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <RootNamespace></RootNamespace>
+    <IsPackable>False</IsPackable>
+    <NoWarn>CS0649;CS0169</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageDownload Include="GitVersion.CommandLine.DotNetCore" Version="[4.0.1-beta1-53]" />
```

Comment:
Why depending on a beta?
It seems `GitVersion.CommandLine.DotNetCore` comes with batteries included.
Can we remove `Lib/`?

Reply:
You want it updated?

Reply:
I would rather rely on a "stable" release if it works.
Mostly I just wondered why `beta1-53`, it seemed so arbitrary to me, as if there was something particular about exactly that version.

Reply:
It just was the latest available at that time I wrote the template.